Sources/armory/logicnode/GetRigidBodyAttributesNode.hx

```diff
@@ -0,0 +1,32 @@
+package armory.logicnode;
+
+import iron.object.Object;
+import armory.trait.physics.RigidBody;
+
+class GetRigidBodyAttributesNode extends LogicNode {
+
+	public function new(tree: LogicTree) {
+		super(tree);
+	}
+
+	override function get(from: Int): Dynamic {
+		var object: Object = inputs[0].get();
+
+		if (object == null) false;
+
+		#if arm_physics
+		var rigidBody = object.getTrait(RigidBody);
+
+		if (from == 0) return rigidBody.group;
```

Comment:
I would use a switch-case statement here to speed things a bit up, like this:
```haxe
return switch (from) {
	case 1: rigidBody.mask;
	case 2: rigidBody.animated;
  ...
  default: null;
}

Reply:
Okay i will fix this too. It is also because i based it in `Get Linear Velocity`. Thanks for the tips!

Reply:
Both ways work, but for larger "sets" of values switch statements are often the better choice as they usually have a O(1) (constant-time) lookup due to a lookup table instead of if-statements where each statement has to be evaluated in order. For only two values it's not a big difference